app/experimenter/experiments/changelog_utils/nimbus.py

```diff
@@ -23,15 +21,12 @@ class Meta:
 
 
 class NimbusExperimentChangeLogSerializer(serializers.ModelSerializer):
+    parent = serializers.SlugRelatedField(read_only=True, slug_field="slug")
     reference_branch = NimbusBranchChangeLogSerializer()
     branches = NimbusBranchChangeLogSerializer(many=True)
     feature_config = NimbusFeatureConfigChangeLogSerializer()
-    owner = serializers.SlugRelatedField(
-        queryset=get_user_model().objects.all(), slug_field="email"
-    )
-    projects = serializers.SlugRelatedField(
-        many=True, queryset=Project.objects.all(), slug_field="slug"
-    )
+    owner = serializers.SlugRelatedField(read_only=True, slug_field="email")
```

Comment:
Had a little head-scratch at this change, but makes sense after I read the DRF docs about read only fields

Reply:
Oh yeah exactly I saw that in the docs and realized we didn't need to pass qs at all 👍